src/main/java/picard/fingerprint/ExtractFingerprint.java

```diff
@@ -82,6 +82,11 @@ public class ExtractFingerprint extends CommandLineProgram {
     @Argument(doc = "When true code will check for readability on input files (this can be slow on cloud access)")
     public boolean TEST_INPUT_READABILITY = true;
 
+
+    @Hidden
```

Comment:
1.  default=true boolean arguments are very strange to me.  Why not invert to "EXTRACT_ALL_VARIANTS" or something like that?

2.  Why is it hidden?  Would Advanced make more sense?

Reply:
sure.